content/WebAuthn/WebAuthn_Walk-Through.adoc

```diff
@@ -0,0 +1,283 @@
+
+
+=== Prerequisites
+You will need at least one YubiKey, plus installation of the following:
+
+Yubico’s Java WebAuthn Server demo:: link:https://github.com/Yubico/java-webauthn-server/tree/master/webauthn-server-demo[webauthn-server-demo]
+Git:: link:https://git-scm.com/[https://git-scm.com/]
+Dependency configuration:: link:https://github.com/Yubico/java-webauthn-server#dependency-configuration[dependency-configuration]: Maven or Gradle
+Javadoc API documentation:: link:https://github.com/Yubico/java-webauthn-server/JavaDoc/webauthn-server-core/latest/com/yubico/webauthn/package-summary.html[package-summary] for the following:
+
+    * ``CredentialRepository`` interface,
+    * ``RelyingParty`` class and
+    * ``navigator.credentials.create()`` and
+    * ``navigator.credentials.get()`` methods.
+
+
+=== Resources
+
+* link:https://fidoalliance.org/specs/fido-v2.0-id-20180227/fido-client-to-authenticator-protocol-v2.0-id-20180227.html#authenticator-api[Client to Authenticator Protocol (CTAP) authenticator API]
+* link:https://www.w3.org/TR/webauthn/[Web Authentication Public Key Credentials API]
+* link:../Software_Projects/WebAuthn-FIDO2/WebAuthn-FIDO2_Server_Libraries/[WebAuthn FIDO2 Server Libraries]
+* link:../Software_Projects/WebAuthn-FIDO2/WebAuthn-FIDO2_Host_Libraries/[WebAuthn FIDO2 Host Libraries]
+* link:https://www.yubico.com/products/services-software/download/yubikey-manager/[YubiKey Manager]
+
+
+=== Run Demo
+To start the demo server, run the following in the ``java-webauthn-server`` directory:
+....
+./gradlew run
+....
+When the server is running, use a browser supporting WebAuthn and open https://localhost:5000 to access the website.
+
+[NOTE]
+======
+You will get warnings in your browser about the connection not being secure. This is expected, because this server uses a self-signed certificate. You can safely proceed to the site.
+======
+
+
+=== Registration
+The sequence of events in the registration flow is described in detail in the “Registration flow” section of the link:WebAuthn_Developer_Guide/WebAuthn_Client_Registration.adoc[WebAuthn Client Registration chapter of the WebAuthn Developer Guide].
+
+
+=== Authentication
+The sequence of events in the authentication flow is described in detail in the “Authentication Flow” section of the link:/WebAuthn_Developer_Guide/WebAuthn_Client_Authentication.adoc[Client Authentication chapter of the WebAuthn Developer Guide].
+
+
+== Client-side Development
+
+=== Configure Client Registration Ceremony
+
+*1 Client initiates request to register an authenticator* User initiates account creation, or initiates new credential registration in account settings for an existing account.
```

Comment:
My comment for the remainder of this section of text is to transform it to be more of a narrative. The sentences are very short and concise which makes if feel a bit choppy as I read through it. I suggest sprinkling some of the why's behind the actions being taken.